configure

```diff
@@ -795,8 +795,8 @@ void foo(void) {
 EOF
 
   if try $CC -march=armv8-a+crc $CFLAGS $test.c -c $test; then
-    CFLAGS="-march=armv8-a+crc $CFLAGS"
-    SFLAGS="-march=armv8-a+crc $SFLAGS"
+    CFLAGS="-march=armv8-a+crc -DADLER32_SIMD_NEON $CFLAGS"
```

Comment:
The library assumes NEON is always available and won't run without it, so I guess this flag could be avoided

Reply:
Guess you need it to avoid breaking x86 though